package.json

```diff
@@ -109,7 +109,7 @@
     "fastify-plugin": "^4.5.1",
     "flat": "^5.0.2",
     "helmet": "^7.1.0",
-    "holocron": "^1.10.0",
+    "holocron": "1.10.1",
```

Comment:
this version includes a bug which allows us to provide redux and react-redux as peer deps.